numpy/ma/extras.py

```diff
@@ -579,8 +579,8 @@ def average(a, axis=None, weights=None, returned=False):
     >>> avg, sumweights = np.ma.average(x, axis=0, weights=[1, 2, 3],
     ...                                 returned=True)
     >>> avg
-    masked_array(data=[2.6666666666666665, 3.6666666666666665],
-                 mask=[False, False],
+    masked_array(data=[2.66666667, 3.66666667],
+                 mask=False,
```

Comment:
The mask is now reduced. It came in as `False`, so I'm not sure if it should be broadcast up to a filled boolean array here or not. It seems like it should be just `mask=False` to me.